odbc/src/api/functions.rs

```diff
@@ -1785,11 +1785,441 @@ pub unsafe extern "C" fn SQLGetInfo(
+                | SQL_FN_TSI_YEAR;
+            return i16_len::set_output_fixed_data(&TIMEDATE_INTERVALS, info_value_ptr, string_length_ptr);
+        }
+        // SQL_CATALOG_LOCATION
+        InfoType::CatalogLocation => {
+            // MongoSQL puts the catalog (database) at the start of a qualified
+            // table name. As in, db.table.
+            return i16_len::set_output_fixed_data(&SQL_CL_START, info_value_ptr, string_length_ptr);
+        }
+        // SQL_SQL_CONFORMANCE
+        InfoType::SqlConformance => {
+            // MongoSQL is SQL-92 Entry level compliant.
+            return i16_len::set_output_fixed_data(
+                &SQL_SC_SQL92_ENTRY,
+                info_value_ptr,
+                string_length_ptr,
+            );
+        }
+        // SQL_ODBC_INTERFACE_CONFORMANCE
+        InfoType::OdbcInterfaceConformance => {
+            // The MongoSQL ODBC Driver is currently meets the minimum compliance level.
+            return i16_len::set_output_fixed_data(&SQL_OIC_CORE, info_value_ptr, string_length_ptr);
+        }
+        // SQL_SQL92_PREDICATES
+        InfoType::Sql92Predicates => {
+            // MongoSQL supports the following SQL-92 predicate operators.
+            const PREDICATES: u32 = SQL_SP_EXISTS
+                | SQL_SP_ISNOTNULL
+                | SQL_SP_ISNULL
+                | SQL_SP_LIKE
+                | SQL_SP_IN
+                | SQL_SP_BETWEEN
+                | SQL_SP_COMPARISON
+                | SQL_SP_QUANTIFIED_COMPARISON;
+            return i16_len::set_output_fixed_data(&PREDICATES, info_value_ptr, string_length_ptr);
+        }
+        // SQL_SQL92_RELATIONAL_JOIN_OPERATORS
+        InfoType::Sql92RelationalJoinOperators => {
+            // MongoSQL supports the following SQL-92 JOIN operators.
+            const JOIN_OPS: u32 = SQL_SRJO_CROSS_JOIN
+                | SQL_SRJO_INNER_JOIN
+                | SQL_SRJO_LEFT_OUTER_JOIN
+                | SQL_SRJO_RIGHT_OUTER_JOIN;
+            return i16_len::set_output_fixed_data(&JOIN_OPS, info_value_ptr, string_length_ptr);
+        }
+        // SQL_AGGREGATE_FUNCTIONS
+        InfoType::AggregateFunctions => {
+            // MongoSQL supports the following aggregate functions.
+            const AGG_FUNCTIONS: u32 = SQL_AF_AVG
+                | SQL_AF_COUNT
+                | SQL_AF_MAX
+                | SQL_AF_MIN
+                | SQL_AF_SUM
+                | SQL_AF_DISTINCT
+                | SQL_AF_ALL;
+            return i16_len::set_output_fixed_data(&AGG_FUNCTIONS, info_value_ptr, string_length_ptr);
+        }
+        // SQL_RETURN_ESCAPE_CLAUSE
+        InfoType::ReturnEscapeClause => {
+            // MongoSQL has not support for the return escape clause since it is read-only.
+            return i16_len::set_output_fixed_data(&SQL_U16_ZERO, info_value_ptr, string_length_ptr);
+        }
+        // SQL_API_SQLFETCHSCROLL
+        InfoType::ApiSqlFetchScroll => {
```

Comment:
This is a function id for https://learn.microsoft.com/en-us/sql/odbc/reference/syntax/sqlgetfunctions-function, that's why it is not documented for SQLGetInfo. 
It an error on my part copy-pasting the list of properties from the extracted flow.
You can remove it :)

Reply:
Removed! thanks